src/index.html

```diff
@@ -3,82 +3,90 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="A project that hightlights a11y fails, created by @5t3ph">
     <link
       href="https://fonts.googleapis.com/css2?family=Rowdies:wght@700&display=swap"
       rel="stylesheet"
     />
     <link rel="stylesheet" href="/css/style.css" />
+    <title>A11y Fails</title>
   </head>
   <body>
-    <ul class="nav">
-      <li><a href="/about">About</a></li>
-      <li><a href="/contact">Contact</a></li>
-      <li><a href="/products">Products</a></li>
-    </ul>
-    <div class="header">
+    <a href="#main" class="skip-link">Skip to main content</a>
+    <nav role="navigation">
```

Comment:
The `role` is redundant in this case 👍 Use of `nav` is sufficient to indicate the role.

Reply:
Ok thank you! I've seen this redundant usage around the web so I'm glad to understand now that landmark elements like `<nav>` natively have the role assigned. I made sure to remove this with commit `129dbf1`